compiler/parser/flex-chapel.cpp

```diff
@@ -3135,7 +3135,7 @@ static char* eatStringLiteral(yyscan_t scanner, const char* startChar) {
       ParserContext context(scanner);
 
       yyText[0] = '\0';
-      yyerror(yyLloc, &context, "end-of-line in a string literal without a preceeding backslash");
+      yyerror(yyLloc, &context, "end-of-line in a string literal without a preceding backslash");
```

Comment:
Note that this is a generated file.  Did you run "make parser" in the parser directory to get these changes?